packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -1527,9 +1523,8 @@ export interface VideoBackgroundNoEffect {
 }
 
 // @beta
-export interface VideoBackgroundReplacementEffect {
-    backgroundImageUrl: string;
-    effectKey: string;
+export interface VideoBackgroundReplacementEffect extends BackgroundReplacementConfig {
+    effectKey?: string;
```

Comment:
The type is VideoEffect
Do we still need name it like effectKey? Maybe just key?
effectName sounds like an effect type, cause it will be one key for one name, but this one feels one name for multiple keys

Reply:
I have updated the `effectkey` to `key`. We need the key for replacement effect to show the highlighted  tile of selected effect and to be able to apply the effect when camera is toggled.